src/SmartEnum/TValue.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Ardalis.SmartEnum
+{
```

Comment:
This class is not used. It can be removed.

Reply:
Agreed. Removed.